README.md

```diff
@@ -5,3 +5,7 @@ Hi there , lets get up
 rahul
 kunal
 utkarsh
```

Comment:
naming issue
------------------------------------------------
ISSUE CODE:
JS001 - Variable naming convention should be correct